docker/reverse_proxy/www/map.html

```diff
@@ -0,0 +1,21 @@
+<html lang="en">
+<head>
+    <script src='https://api.mapbox.com/mapbox-gl-js/v2.1.0/mapbox-gl.js'></script>
+    <link href='https://api.mapbox.com/mapbox-gl-js/v2.1.0/mapbox-gl.css' rel='stylesheet' />
+    <title>Ehrenamtskarte Preview Map</title>
+</head>
+<body>
+<div id='map' style='width: 100%; height: 100%;'></div>
+<script>
+  mapboxgl.accessToken = 'pk.eyJ1IjoibWF4YW1tYW5uIiwiYSI6ImNraHlzdGNmejBkaGsycm9hNjJjcWZzdmwifQ.3NaekT5CjDsBnBNdUr1F7g';
```

Comment:
whoopsie?

Reply:
Noopsie just an alreadly public key which is not used. Dummy key.